### DISCUSSION TITLE
Dokan time slot

### DISCUSSION THREAD

**Discussion by kuwaitman** [inspireui/support#5223] [New Feature/Ideas Requests]
hi i was wondering if you are planning to add Dokan delivery time slot module integration
https://wedevs.com/docs/dokan/modules/dokan-delivery-time/
to:
https://codecanyon.net/item/fluxstore-multi-vendor-flutter-ecommerce-full-app/25013325/

Kind Regards,

**Comment by Dokanuser** [comment]
Also need the delivery time being displayed on fluxstore manager for Dokan urgently as vendors need to see when to be ready. Please add this as soon as possible
